src/api/spec/models/kiwi/image_spec.rb

```diff
@@ -76,4 +84,159 @@
       it { expect(subject.valid?).to be_falsey }
     end
   end
+
+  describe '.to_xml' do
```

Comment:
Should be `#to_xml` since its an instance method:

http://www.betterspecs.org/#describe